cmd/age-keygen/age-keygen.1

```diff
@@ -0,0 +1,25 @@
+.Dd June 28, 2020
+.Dt AGE-KEYGEN 1
+.Sh NAME
+.Nm age-keygen
+.Nd generate an age secret key
+.Sh SYNOPSIS
+.Nm
+.Op Fl o Ar file
+.Sh DESCRIPTION
+.Nm
+generates an
+.Nm age
+secret key.
+.Pp
+If the
+.Fl o
+flag is not specified the key is output to standard output with the public key
```

Comment:
Grammar nit: a comma after “specified” is required here due to the prefixed “if” clause as far as I know.